framework/src/controller/bus.ts

```diff
@@ -97,6 +98,7 @@ export class Bus {
 		this._events = {};
 		this._channels = {};
 		this._rpcClients = {};
+		this._context = {};
```

Comment:
I think context changes on every call, so it's better not to keep the context as member. Also, this is never updated, so I think not needed?

Reply:
Yes, it's really not needed. Need to pass context in other functions that call the invoke() method.